resources/recipes.py

```diff
@@ -674,9 +674,9 @@ def item_tag(namespace: str, _variant: str) -> str:
             anvil_recipe(rm, '%s_sheet' % metal, item_tag('forge', 'double_ingot'), item('sheet'), metal_data.tier, Rules.hit_last, Rules.hit_second_last, Rules.hit_third_last)
             anvil_recipe(rm, '%s_rod' % metal, item_tag('forge', 'ingot'), '2 tfc:metal/rod/%s' % metal, metal_data.tier, Rules.bend_last, Rules.draw_second_last, Rules.draw_third_last)
 
-        # Toolss
+        # Tools
         if 'tool' in metal_data.types:
-            anvil_recipe(rm, '%s_tuyere' % metal, '#forge:double_sheets/%s' % metal, 'tfc:metal/tuyere/%s' % metal, metal_data.tier, Rules.bend_last, Rules.bend_second_last)
+            anvil_recipe(rm, '%s_tuyere' % metal, '#forge:double_sheet/%s' % metal, 'tfc:metal/tuyere/%s' % metal, metal_data.tier, Rules.bend_last, Rules.bend_second_last)
```

Comment:
Yea, Got confused a bit, Had an issue where it would gen tags with 2 's'. Got it sorted now.

As to why it's not a function, I was just following the pre-existing code. Looking at the unmodified file, It's the same.